pkg/util/testing/wrappers.go

```diff
@@ -102,7 +102,11 @@ func (w *WorkloadWrapper) Queue(q string) *WorkloadWrapper {
 	return w
 }
 
-// ReserveQuota sets workload admission and adds a "QuotaReserved" status condition
```

Comment:
Can you move this comment to Line 109 instead of removing it?

Reply:
ohhh my bad, done